pkg/utils/utils.go

```diff
@@ -150,3 +151,17 @@ func ParsePKey(pKey string) (int, error) {
 
 	return int(i), nil
 }
+
+// ParseNetworkId returns the network name and network namespace
+func ParseNetworkId(networkId string) (string, string, error) {
```

Comment:
separator should be `_`
namespace and network are valid FQDN names they both may have .

see:
```
# kubectl apply -f /tmp/namespace.yaml
The Namespace "my_namespace" is invalid: metadata.name: Invalid value: "my_namespace": a DNS-1123 label must consist of lower case alphanumeric characters or '-', and must start and end with an alphanumeric character (e.g. 'my-name',  or '123-abc', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?')
```

Reply:
"." is not allowed in namespaces
```
[root@r-cloudx3-07 Mellanox]# kubectl create namespace ib.kubernetes
The Namespace "ib.kubernetes" is invalid: metadata.name: Invalid value: "ib.kubernetes": a DNS-1123 label must consist of lower case alphanumeric characters or '-', and must start and end with an alphanumeric character (e.g. 'my-name',  or '123-abc', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?')
```

Reply:
is it also not allowed in network name ?

Reply:
if so im OK

Reply:
seems "." is allowed in network CRD name but "_" is not allowed, so changed the separator to "_"